tmt/schemas/core.yaml

```diff
@@ -21,6 +21,10 @@ definitions:
   description:
     type: string
 
+  # https://tmt.readthedocs.io/en/stable/spec/core.html#contact
+  contact:
+    $ref: "/schemas/common#/definitions/one_or_more_strings"
+
```

Comment:
The `definitions` are sorted here alphabeticaly. Let's move this one item up to keep the order.

Reply:
Sure, updated.